dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_chart_options.tsx

```diff
@@ -44,56 +44,78 @@ export const ConfigChartOptions = ({
     return (
       currentSchemas &&
       currentSchemas.map((schema, index) => {
-        let params = {};
+        let params = {
+          title: schema.name,
+          vizState,
+          ...schema.props,
+        };
         const DimensionComponent = schema.component || PanelItem;
         if (schema.eleType === 'palettePicker') {
```

Comment:
Could we consider to optimize code here for this set of long if/else branches, maybe with switch?

Reply:
Sure, Thanks @mengweieric, I will do these changes in a separate PR for sprint2.
Created an issue for the same here: https://github.com/opensearch-project/observability/issues/906